test/unittest/tests/block/test_block_other.cpp

```diff
@@ -19,27 +19,23 @@
 static void
 test_block_header_byteswap_copy(WT_BLOCK_HEADER *from, WT_BLOCK_HEADER *to)
 {
-    WT_BLOCK_HEADER prev_from, prev_to;
```

Comment:
We were hitting a compile error with `WORDS_BIGENDIAN=1` because this field is set but never used.
Details on the other changes are called out below